Makefile

```diff
@@ -12,14 +14,21 @@ THIS_MAKEFILE_PATH := $(word $(words $(MAKEFILE_LIST)),$(MAKEFILE_LIST))
 THIS_DIR := $(shell cd $(dir $(THIS_MAKEFILE_PATH));pwd)
 NPM_BIN_DIR = $(shell npm bin)
 
+NPMRC_ELECTRON_VERSION := $(shell npm config get target)
+PACKAGE_ELECTRON_VERSION := $(shell node -e "const electronVersion = require('./package.json').devDependencies.electron; console.log( electronVersion )")
```

Comment:
might as well simplify here…

```js
node -e "console.log(require('./package.json').devDependencies.electron)"
```

…that is, there's no value in creating the middleman variable

Reply:
Good suggestion - refactored in a99c72b